go/mysql/binlog.go

```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2015 Shlomi Noach, courtesy Booking.com
+   Copyright 2021 GitHub Inc.
```

Comment:
Technically you're not supposed to remove this copyright. You're only allowed to append more copyrights. This piece of code was imported from `orchestrator` code that I authored while working at Booking.com.

Reply:
@shlomi-noach ahah, mass copy+paste error!

I don't think we have a preference, my aim was to update the year to reflect this code is still being actively developed.

Would `2021 Shlomi Noach, courtesy Booking.com` be more correct or just leave as-is?

Reply:
leave original 2015 row as it is, and, if you like, append new 2021-github row